ocaml/idl/datamodel_host.ml

```diff
@@ -1029,6 +1029,14 @@ let host_query_ha = call ~flags:[`Session]
       ~allowed_roles:_R_READ_ONLY
       ()
 
+  let refresh_server_certificates = call
+      ~lifecycle:[Published, rel_next, ""]
+      ~name:"refresh_server_certificates"
+      ~doc:"Replace the self-signed certficates for the host with new ones."
+      ~params:[Ref _host, "host", "The host"]
+      ~allowed_roles:_R_POOL_ADMIN
```

Comment:
This should not be documented, IMO. Users will find confusing having "refresh_server_certificates" and "reset_server_certificate".
I think recycling certificates in this way only makes sense for internal certificates, dropping the 's' among other changes would make it less confusing even for us.

Reply:
Ok - will limit this to the internal certificates. But to be clear: a server has two self-signed certificates, one for internal and one for external use. So don't rotate the external one? I don't think this makes sense from a security perspective.

Reply:
The external one is already handled by the call `reset_server_certificate`. It's a different call because users might have installed their own certificates and they just want to replace the self-signed ones without affecting the external certificates.